ravenframework/Optimizers/GeneticAlgorithm.py

```diff
@@ -41,8 +41,7 @@
 from .crossOverOperators.crossovers import returnInstance as crossoversReturnInstance
 from .mutators.mutators import returnInstance as mutatorsReturnInstance
 from .survivorSelectors.survivorSelectors import returnInstance as survivorSelectionReturnInstance
-from .survivorSelectors import survivorSelectors 
-from .survivorSelectors import survivorSelectors 
+from .survivorSelection import survivorSelection as survivorSelectionProcess 
```

Comment:
So we have a folder called survivorSelectors and another called survivorSelection? Or did you mean to remove one?